src/stlink-lib/chipid.c

```diff
@@ -332,6 +332,8 @@ static const struct stlink_chipid_params devices[] = {
         .sram_size = 0x2000,                   // "SRAM" byte size in hex from Table 2
         .bootrom_base = 0x1fffec00,            // "System memory" starting address from Table 2
         .bootrom_size = 0xC00,                 // "System memory" byte size in hex from Table 2
+        .option_base = STM32_F0_OPTION_BYTES_BASE,
```

Comment:
This can be set similarly for all the F0 and F1 series. The F3 also, but the flash process may need to be improved

Reply:
Then we should name and use this define appropriately from my point of view.